src/containers/Sidebar/SidebarBottomItem.js

```diff
@@ -51,11 +51,31 @@ const SidebarBottomItem = ({ item }) => {
 								title={t(`Sidebar|${item.name}`)}
 								className={`sidebar-item-button${location.pathname === item.url ? " active " : " "}btn left`}
 							>
-								<Icon icon={item.icon} />
-								<div className="sidebar-item-name ml-2">{t(`Sidebar|${item.name}`)}</div>
-								{item.children && item.children.length > 0 && (
-									<Icon icon={isOpen ? "cil-arrow-circle-bottom" : "cil-arrow-circle-left"} />
-								)}
+								{sidebarLogo ? 
+									isSidebarCollapsed ? 
+										<img
+											src={sidebarLogo?.minimized}
+											alt={t(`About`)}
+											width="30"
+											height="30"
+										/>
+									:
+										<img
+											src={sidebarLogo?.full}
+											alt={t(`About`)}
+											width="85"
+											height="30"
+										/>
+								:
+									<>
+										<Icon icon={item.icon} />
+										<div className="sidebar-item-name ml-2"> {t(`Sidebar|About`)}</div>
```

Comment:
@petrKavulok Here should be: ```<div className="sidebar-item-name ml-1">{t(`Sidebar|${item.name}`)}</div>``` -- although not sure about the `ml-1` as I told you about the margin with other Sidebar buttons